.gitmodules

```diff
@@ -40,3 +40,6 @@
 [submodule "cortx-halo"]
 	path = cortx-halo
 	url = https://github.com/Seagate/cortx-halo.git
+[submodule "cortx-cc"]
```

Comment:
https://github.com/Seagate/cortx-cc is still private. I feel we should not add it here.

Reply:
fix as though I didnt add this code